src/index.html

```diff
@@ -2,9 +2,15 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
     <title>Country Information - World Regions</title>
 </head>
 <body>
+<h1>World Regions</h1>
+<img src="./assets/world_map.png" alt="World Map" max width="100%">
```

Comment:
css moet in het losse .css bestand, niet inline in je html. Dus geen `max width="100%"` maar een `class` hier aanmaken en dit dan in de .css zetten.